plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -45,36 +65,42 @@ module.exports = ({containerConfig, appConfig}) => {
         <ConfigurationMapLabel>Container Image</ConfigurationMapLabel>
         <ConfigurationMapValueWithDefault
           value={findNestedPropertyInObject(appConfig, 'image.id')} />
+        {editLink}
```

Comment:
Could this justify the creation of a `<ConfigurationMapEditLink>` component? If you also follow my suggestion for passing down the handleEdit callback, this could be more cleanly defined as:

```jsx
<ConfigurationMapEditLink
   section="general"
   field="image"
   onClick={this.props.onEditClick} />
```